.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "files.maxMemoryForLargeFilesMB": 6096
+}
```

Comment:
Please don't track .vscode directory. Remove it by: git rm -r --cached <file>
After that action please double check if you have .vscode/ in gitignore.

Reply:
Got it. I made corrections.